docs/commands-guide.md

```diff
@@ -78,8 +78,7 @@ This is a guide to all of the slash commands for the app.
     ```
     /message-stream stream true
     ```
-    > [!NOTE]
-    > This is a very slow progress on Discord because "spamming" changes within 5 seconds is not allowed.
+    **This is a very slow progress on Discord because "spamming" changes within 5 seconds is not allowed.**
```

Comment:
I'm criticizing my own sentence but probably "This is very slow on Discord because "spamming" changes in a channel within a period of 5 seconds is not allowed."

Reply:
Corrected.